lib/taro/types/field.rb

```diff
@@ -30,6 +30,10 @@ def openapi_type
     type.openapi_type
   end
 
+  def openapi_format
+    type.openapi_format
```

Comment:
:+1: side note: atm, enum is the only property that can be specified both on types and on individual fields. it might be worth thinking about whether the same should be possible for other specifiers like format or pattern, or rather not? but this decision is no blocker for this PR.

Reply:
True, I think it makes sense to add it once the first use case comes up?